src/workers/answerQuestion.ts

```diff
@@ -47,21 +47,33 @@ const worker = new Worker(
           },
         ],
         false,
-        256
+        512,
+        'open-mixtral-8x22b'
```

Comment:
Attempt to use model with 64k context instead of just 32k for answering with data.